python-avocado.spec

```diff
@@ -148,6 +148,7 @@ popd
 %{__mkdir} -p %{buildroot}%{_mandir}/man1
 %{__install} -m 0644 man/avocado.1 %{buildroot}%{_mandir}/man1/avocado.1
 %{__install} -m 0644 man/avocado-rest-client.1 %{buildroot}%{_mandir}/man1/avocado-rest-client.1
+%{__install} -d -m 0755 %{buildroot}%{_sharedstatedir}/avocado/data
```

Comment:
As usual I'm not certain by this as I have little experience with SPEC files, but I read that the `_sharedstatedir` translates to `/var/lib` on RHEL/Fedora and on intel-based systems, but other distros or architectures can define it as something else.

Not sure how this is treated by other SPECs, but probably there should be install script which adjusts the `base` and `data` dirs in config file...

anyway I'll believe you if you say it's uncommon enough to ignore it, just raising a question here.